frontend/src/bundles/admin/verification/pages/verification.tsx

```diff
@@ -0,0 +1,418 @@
+import { ManageSearch } from '@mui/icons-material';
+import { Grid } from '@mui/material';
+import { useCallback } from 'react';
+import { type ValueOf } from 'shared/build/index.js';
+
+import {
+    Avatar,
+    Button,
+    Typography,
+} from '~/bundles/common/components/components.js';
+import { IconButton } from '~/bundles/common/components/icon-button/icon-button.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+import {
+    useEffect,
+    useMediaQuery,
+    useState,
+    useTheme,
+} from '~/bundles/common/hooks/hooks.js';
+
+import { PreviewTabs } from '../constants/constants.js';
+import { type FilterValues } from '../types/filter-values.types.js';
+import { VerificationList } from './../components/components.js';
+import { employers, talents } from './../mock-data/mock-data.js';
+import styles from './styles.module.scss';
+
+const Verification: React.FC = () => {
+    const [filter, setFilter] = useState<FilterValues>('talents');
+    type TabValues = ValueOf<typeof PreviewTabs>;
+    const items = filter === 'talents' ? talents : employers;
+    const FIRST_INDEX = 0;
+    const theme = useTheme();
+
+    const [isFilterOpen, setIsFilterOpen] = useState<boolean>(false);
+    const [selectedId, setSelectedId] = useState<string>(
+        items[FIRST_INDEX].userId,
+    );
```

Comment:
When talents or employers array is empty page crashes trying to get userId of undefined